code for tinkering audio/audio_Envelops_And_Echos.py

```diff
@@ -0,0 +1,17 @@
+#audio envelops and echos.
```

Comment:
Use [doc-strings](https://www.python.org/dev/peps/pep-0257/#what-is-a-docstring) where appropriate.